ingenialink/ethercat/network.py

```diff
@@ -95,18 +102,36 @@ def __init__(
         self._ecat_master: pysoem.CdefMaster = pysoem.Master()
         self._ecat_master.sdo_read_timeout = int(1_000_000 * connection_timeout)
         self._ecat_master.sdo_write_timeout = int(1_000_000 * connection_timeout)
+        self._ecat_master.manual_state_change = self.MANUAL_STATE_CHANGE
         self.__is_master_running = False
+        self.__last_init_nodes: List[int] = []
 
     def scan_slaves(self) -> List[int]:
-        """Scans for nodes in the network.
+        """Scans for nodes in the network. If any node is already connected scan can not be done.
```

Comment:
I think "node" is ok, canopen communication has the same docstring.
About "The scanning can only be done before connecting to a slave.", you can scan again if you disconnect the node, so you can do it after connecting a slave if you disconnect it.

Reply:
I think node is CAN terminology and slave is EtherCAT terminology, they don't need to be the same. 
Of course, if you disconnect you can scan again. Maybe it should say: "Scanning of slaves cannot be done if a slave is already connected to the network."

Reply:
In EtherCAT node and slave can be used indistinctly for all the nodes except the master, I prefer to keep it as node to keep a similar docstring as CANopen, because the functionality of this function is completely equivalent.